src/services/JobRoleService.ts

```diff
@@ -1,5 +1,6 @@
+import axios, {AxiosResponse} from "axios"
```

Comment:
Import as a type to save the polar bears

Reply:
Done 
<img width="662" alt="image" src="https://github.com/user-attachments/assets/d4ee27a5-0712-4749-a7c1-f07e01d9bb61">